test/integ/pandas/test_pandas_tools.py

```diff
@@ -142,6 +152,8 @@ def test_write_pandas_with_overwrite(
 @pytest.mark.parametrize("quote_identifiers", [True, False])
 @pytest.mark.parametrize("auto_create_table", [True, False])
 @pytest.mark.parametrize("create_temp_table", [True, False])
+@pytest.mark.parametrize("index", [True, False])
+@pytest.mark.parametrize("engine", ["auto", "pyarrow"])
 def test_write_pandas(
```

Comment:
Same as the other comment. However; this one runs 8 times already and your change would make it run 32 times.

Reply:
I'm going to remove the parametrization on the `engine` arg as `auto` uses `pyarrow` by default. It looks like for the `index` arg there's no incentive to set it to True as the index is saved by default in the metadata in a faster way when left as None, but I think setting it to False could be heavily used to save space for highly transformed DataFrames, so I would like to leave the test for it there.